src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -556,21 +556,22 @@ ${f.resolution}
     return JSON.stringify(tourFindings, null, 2);
   }
 
-  private getLineToModify(finding: FindingToReport): number {
+  private getLineToModify(finding: FindingToReport, rootPath: string): number {
 
-    // Don't cause an issue if the file isn't there
-    if (!fs.existsSync(finding.file)) {
+    const filePath: string = path.resolve(path.join(rootPath, finding.file));
+  
+    // // Don't cause an issue if the file isn't there
+    if (!fs.existsSync(filePath)) {
       return 1;
     }
 
-    // Read the file content
-    const fileContent: string = fs.readFileSync(finding.file, 'utf-8');
-
+    // // Read the file content
+    const fileContent: string = fs.readFileSync(filePath, 'utf-8');
+  
     // Try to find the line this relates to
-    const splits: string[] = fileContent.split(os.EOL);
+    const splits: string[] = fileContent.split('\n'); // os.EOL doesn't work here
```

Comment:
Thanks for checking. It could be that the original project was scaffolded with *nix line endings. Perhaps using `\n` for split is enough on all OSes in this case.